### PR TITLE
Fix compile issue

### PR DESCRIPTION
**Pull request by NoSloppy**
I don't think this was right.

### CODE REVIEW

File: props/saber_shtok_buttons.h [hunk 1]
```diff
@@ -144,7 +144,7 @@ SaberShtokButtons() : PropBase() {}
 // Multi-Blaster Deflection mode
       case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_ON | BUTTON_POWER):
         swing_blast_ = true;
-        hybrid_font.SB_Blast();
+        hybrid_font.DoBlast();
```
Comment (profezzorn) [inline props/saber_shtok_buttons.h:147]: This is not right.
Either we call SaberBase::DoBlast() to notify all saber bases of a blast, or we call hybrid_font.SB_Effect(EFFECT_BLAST, (200 + random(700))/1000.0f)); to only play the blast sound and nothing else.

Comment (NoSloppy) [reply props/saber_shtok_buttons.h:147]: It's Dmitry's prop. I just came across it helping someone out. It's SaberBase::DoBlast(); according the general consensus in the other props, so I'll do that...for him?

Comment (profezzorn) [reply props/saber_shtok_buttons.h]: It's better than not compiling...

Comment (NoSloppy) [reply props/saber_shtok_buttons.h]: Well, turns out it should have been hybrid_font.SB_Effect(EFFECT_BLAST, 0);
When entering the multiblast mode, it just plays the sound.
He has his own pull request in for updates to this file anyway and it's correct in that version, so.....

**Pull request merged**